update

```diff
@@ -139,6 +139,7 @@ async function makeEntry (dir, parsedFileName, oldList) {
     const fileContent = await readFile(absolutePath)
     build.keccak256 = '0x' + ethUtil.keccak(fileContent).toString('hex')
     console.log("Computing hashes of '" + pathRelativeToRoot + "'")
+    build.sha256 = '0x' + ethUtil.sha256(fileContent).toString('hex')
```

Comment:
Can you also modify line 137 accordingly, please?

Reply:
@chriseth done. Mark as resolved if it's well now :)